go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golang/mock v1.4.0
-	github.com/golang/protobuf v1.4.0 // indirect
+	github.com/golang/protobuf v1.4.2 // indirect
```

Comment:
@bentranter is this version bump okay?

Reply:
Yup, these look okay – I just realized some of these were pulled in as a result of different godo changes than yours.